main.js

```diff
@@ -0,0 +1,73 @@
+import $ from 'jquery'
+import './style/style.sass'
+import './style/reset.css'
+import { Todo } from './Todo.js'
+
+const inputText = $('.input-text')
+const buttonAddTodo = $('.add-item')
+const todoList = $('.todo__list')
+const filtrAll = $('.filtr-all')
+const filtrTodo = $('.filtr-todo')
+const filtrDone = $('.filtr-done')
+
+buttonAddTodo.click(() => {
+  const todo = new Todo(inputText.val())
+
+  const createTodo = todo.createNewTodo(inputText.val())
+    .appendTo(todoList)
+
+  const allButton = todo.createAllButton()
```

Comment:
looks better, but you CAN create all elements in the class, just don't append it to the DOM there. Return all HTML related to this todo and append it here